sirepo/raydata/scan_monitor.py

```diff
@@ -657,12 +676,15 @@ def _scan_info(
             PKDict(rduid=rduid, **req_data)
         ).has_analysis_pdfs(),
     )
-    for c in _DEFAULT_COLUMNS.keys():
-        d[c] = getattr(m, c)()
-
-    for c in req_data.get("selectedColumns", []):
-        d[c] = m.get_start_field(c, unchecked=True)
-
+    for c in itertools.chain(
+        _default_columns(req_data.catalogName).keys(),
+        req_data.get("selectedColumns", []),
+    ):
+        d[c] = (
+            getattr(m, c)()
+            if c in _METADATA_COLUMNS
+            else m.get_start_field(c, unchecked=True)
+        )
```

Comment:
Personally I find the old way much clearer

Reply:
I agree. The problem is that now _default_columns contains columns from metadata and start doc so we need a way to handle that. Any suggestions on how to improve?

Reply:
Ah, that makes sense. I think it's good enough then.